test/e2e/openshift/openshift_test.go

```diff
@@ -85,6 +85,62 @@ var _ = Describe("Azure Container Cluster using the OpenShift Orchestrator", fun
 		}
 	})
 
+	It("should whitelist pods on compute nodes", func() {
+		namespaces := []string{
+			"openshift-infra",
+			"openshift-node",
+			"openshift-metrics",
+			"kube-",
+			"default",
+		}
+
+		whitelist := []string{
+			"sync-",
+			"ovs-",
+			"sdn-",
+		}
+		list, err := pod.GetAll("")
+		Expect(err).NotTo(HaveOccurred())
+
+		for _, pod := range list.Pods {
+			skip := false
+			// we have a whitelist of namespaces that we will skip
+			for _, pattern := range namespaces {
+				if strings.Contains(pod.Metadata.Namespace, pattern) {
+					skip = true
```

Comment:
`skip` is ugly - either using Go's `break LABEL` syntax (https://golang.org/ref/spec#Break_statements) or a separate function would be better.

Reply:
why are we excluding certain namespaces?

Reply:
The idea was that any pod that is in the openshift-metrics namespace we want to skip regardless of host.  I see that this might bypass too large a space with the list that I came up with.  I will only skip `openshift-metrics` currently.